src/test/java/simplifier/controllers/UserControllerIT.java

```diff
@@ -96,4 +96,28 @@ public void conflictIfUsernameNotUnique() {
         assertThat(responseEntity.getStatusCode(), is(HttpStatus.CONFLICT));
         assertThat(responseEntity.getBody(), is(requiredMessage));
     }
+
+    @Test
+    public void login() {
```

Comment:
Rename to more informative shouldLoginWithOkStatusAndReturnToken. The name of a test method should  describe expected input and the expected result for that input. Apply this naming to all code